apps/stripe/vitest.config.ts

```diff
@@ -11,20 +11,15 @@ export default defineConfig({
       {
         extends: true,
         test: {
+          sequence: {
+            shuffle: true,
+          },
           include: ["src/**/*.test.ts"],
           exclude: ["src/**/*.integration.test.ts", "src/__tests__/integration/**"], // exclude integration tests so vitest doesn't run them twice
```

Comment:
we will have tests inside folders so `"src/**/*.integration.test.ts"` in exclude is no longer needed

Reply:
done